luxonis_train/__init__.py

```diff
@@ -1,7 +1,8 @@
 from .attached_modules import *
 from .core import *
+from .loaders import *
 from .models import *
 from .nodes import *
+from .optimizers import *
+from .schedulers import *
 from .utils import *
-
-__version__ = "0.0.1"
```

Comment:
It was already in `pyproject.toml` and I thought `__version__` gets populated automatically, but I see that's not the case. Fixed it by pointing the `pyproject.toml` version to this instead to avoid the duplicate version info.